catboost/python-package/catboost/core.py

```diff
@@ -84,6 +85,21 @@ def get_catboost_bin_module():
 _have_equal_features = _catboost._have_equal_features
 SPARSE_MATRIX_TYPES = _catboost.SPARSE_MATRIX_TYPES
 
+_CATBOOST_SKLEARN_COMPAT_TAGS = {
+    'non_deterministic': False,
```

Comment:
this is false for CPU and true for GPU

Reply:
tried to change it in runtime, is this correct?